ovos_padatious/training_manager.py

```
@@ -11,52 +11,70 @@
+from typing import List, Type, Union
+
+from ovos_utils.log import LOG
 
 import ovos_padatious
 from ovos_padatious.train_data import TrainData
 from ovos_padatious.util import lines_hash
 
 
-def _train_and_save(obj, cache, data, print_updates):
-    """Internal pickleable function used to train objects in another process"""
+def _train_and_save(obj: 'Trainable', cache: str, data: TrainData, print_updates: bool) -> None:
```

Ruff

Undefined name `Trainable`

(F821)